stable/redis/README.md

```diff
@@ -80,12 +91,12 @@ The following table lists the configurable parameters of the Redis chart and the
 |--------------------------------------------|----------------------------------------------------------------------------------------------------------------|------------------------------------------------------|
 | `global.imageRegistry`                     | Global Docker image registry                                                                                   | `nil`                                                |
 | `image.registry`                           | Redis Image registry                                                                                           | `docker.io`                                          |
-| `image.repository`                         | Redis Image name                                                                                               | `bitnami/redis`                                      |
+| `image.repository`                         | Redis Image name                                                                                               | `redis`                                              |
```

Comment:
I guess this is `bitnami/redis` again no?

Reply:
good catch, thanks